shared/store/action-logger.js

```diff
@@ -15,13 +22,47 @@ const objToJS = state => {
   return newState
 }
 
-export const actionLogger = store => next => action => {
+const loggingStatSink: StatSink = {
+  label: 'Action Logger',
+  totalTime: 0,
+  totalActions: 0,
+  timings: [], // to calculate variance
+  startTime: 0,
+}
+
+const actionStatSink: StatSink = {
+  label: 'Actions',
+  totalTime: 0,
+  totalActions: 0,
+  timings: [], // to calculate variance
+  startTime: 0,
+}
+
+export const actionLogger = (store: any) => (next: any) => (action: any) => { // eslint-disable-line arrow-parens
   console.groupCollapsed && console.groupCollapsed(`Dispatching action: ${action.type}`)
 
   console.log(`Dispatching action: ${action.type}: ${JSON.stringify(action)} `)
```

Comment:
Stringify kind of makes it less cool in dev tools.. is there a way to not stringify when in DEV mode?